Makefile

```diff
@@ -45,14 +47,16 @@ clean:
 	rm -rf ${RELEASE_SRC}.tgz.sha512
 
 release-src:
-	tar -zcvf $(RELEASE_SRC).tgz \
+	rm -f $(TMPDIR)/$(RELEASE_SRC).tgz && \
```

Comment:
What about move this line to `clean` and let `release-src` depends on `clean`

Reply:
Done